management/schemas/distance-target.yaml

```diff
@@ -0,0 +1,74 @@
+schemas:
+
+  DistanceTarget:
+    type: object
+    properties:
+      Id:
+        type: integer
+        format: int32
+      FlightId:
+        type: integer
+        format: int32
+      Distance:
+        type: number
+        format: float
+        nullable: true
+      Latitude:
+        type: number
+        format: float
+        nullable: true
+      Longitude:
+        type: number
+        format: float
+        nullable: true
+      StreetAddress:
+        type: string
+        nullable: true
+
+  BatchUpload:
+    type: object
+    properties:
+      Distance:
+        type: number
+        format: float
+      Latitude:
+        type: number
+        format: float
+      Longitude:
+        type: number
+        format: float
+
+  BadRequest:
```

Comment:
Unless we're 100% sure about the structure of these, I'd remove them and just leave the `type: object` in the spec portion.

Reply:
Updated